src/libmongoc/doc/mongoc_client_set_timeout_ms.rst

```diff
@@ -0,0 +1,32 @@
+:man_page: mongoc_client_set_timeout_ms
+
+mongoc_client_set_timeout_ms()
+==============================
+
+Synopsis
+--------
+
+.. code-block:: c
+
+  bool
+  mongoc_client_set_timeout_ms (mongoc_client_t *client, int64_t *timeout_ms, bson_error_t *error)
+
+Sets the client-side operation timeout value for ``client`` to the given ``timeout_ms``. If there is an error setting the timeout, this method will return false and ``error`` will be set. If ``client`` belongs to a :symbol:`mongoc_client_pool_t`, its timeout will be reset to match the pool's timeout once it is pushed back into the pool.
```

Comment:
> If ``client`` belongs to a :symbol:`mongoc_client_pool_t`, its timeout will be reset to match the pool's timeout once it is pushed back into the pool.

Is this required by the spec? Also, I don't think we'd need to set timeout MS on a pool.

Reply:
The spec doesn't say anything about pooling, but this behavior can be addressed in https://jira.mongodb.org/browse/CDRIVER-3537 at a later point in time.